tests/integration/targets/openssh_keypair/tasks/main.yml

```diff
@@ -4,509 +4,14 @@
-  ignore_errors: yes
-
-- name: Regenerate broken key with force
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekeybroken'
-    type: rsa
-    force: yes
-    size: 2048
-  register: output_broken_force
-
-- name: Generate read-only private key
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekeyreadonly'
-    type: rsa
-    mode: '0200'
-    size: 2048
-
-- name: Regenerate read-only private key via force
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekeyreadonly'
-    type: rsa
-    force: yes
-    size: 2048
-  register: output_read_only
-
-- name: Generate privatekey7 - standard with comment
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekey7'
-    comment: 'test@privatekey7'
-    size: 2048
-  register: privatekey7_result
-
-- name: Modify privatekey7 comment
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekey7'
-    comment: 'test_modified@privatekey7'
-    size: 2048
-  register: privatekey7_modified_result
-
-- name: Generate password protected key
-  command: 'ssh-keygen -f {{ output_dir }}/privatekey8 -N {{ passphrase }}'
-
-- name: Try to modify the password protected key - should fail
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekey8'
-    size: 2048
-  register: privatekey8_result
-  ignore_errors: yes
-
-- name: Try to modify the password protected key with force=yes
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekey8'
-    force: yes
-    size: 2048
-  register: privatekey8_result_force
-
-- name: Generate another password protected key
-  command: 'ssh-keygen -f {{ output_dir }}/privatekey9 -N {{ passphrase }}'
-
-- name: Try to modify the password protected key with passphrase
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekey9'
-    size: 1024
-    passphrase: "{{ passphrase }}"
-  register: privatekey9_modified_result
-  when: cryptography_version.stdout is version('3.0', '>=') and bcrypt_version.stdout is version('3.1.5', '>=')
-
-- name: Generate another unprotected key
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekey10'
-    size: 2048
-
-- name: Try to Modify unprotected key with passphrase
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekey10'
-    size: 2048
-    passphrase: "{{ passphrase }}"
-  ignore_errors: true
-  register: privatekey10_result
-  when: cryptography_version.stdout is version('3.0', '>=') and bcrypt_version.stdout is version('3.1.5', '>=')
-
-
-- name: Try to force modify the password protected key with force=true
-  openssh_keypair:
-    path: '{{ output_dir }}/privatekey10'
-    size: 2048
-    passphrase: "{{ passphrase }}"
-    force: true
-  register: privatekey10_result_force
-  when: cryptography_version.stdout is version('3.0', '>=') and bcrypt_version.stdout is version('3.1.5', '>=')
-
-- name: Ensure that ssh-keygen can read keys generated with passphrase
-  command: 'ssh-keygen -yf {{ output_dir }}/privatekey10 -P {{ passphrase }}'
-  register: privatekey10_result_sshkeygen
+- set_fact:
```

Comment:
For other modules with multiple backends, there's usually one simple test with backend autodetect before the backend-specific tests, like https://github.com/ansible-collections/community.crypto/blob/main/tests/integration/targets/openssl_privatekey/tasks/main.yml#L34. Basically it only makes sure the module doesn't crash when a backend isn't specified :)